src/test/java/com/spotify/futures/CompletableFuturesTest.java

```diff
@@ -206,6 +208,15 @@ public void getException_cancelled() throws Exception {
     getException(future);
   }
 
+  @Test
+  public void getException_cancelledPropagated() throws Exception {
```

Comment:
Is it expected that propagated cancellation has a different behaviour than direct cancellation?

I think it would be nice if getException handled them the same way

Reply:
Thanks for bringing this up and providing a testcase!
Agree with you, I think this should behave the same way as `getException_cancelled` above.